src/libexpr/primops.cc

```diff
@@ -4133,26 +4268,29 @@ void EvalState::createBaseEnv()
         attrs.alloc("prefix").mkString(i.first);
         (v.listElems()[n++] = allocValue())->mkAttrs(attrs);
     }
-    addConstant("__nixPath", v);
+    addConstant("__nixPath", v, {
+        .type = "list",
+        .doc = R"(
+          The current 'Nix Path' search path.
+        )",
+    });
```

Comment:
```suggestion
    addConstant("__nixPath", v);
```

We should not document this at all

Reply:
`builtins.nixPath` exists and can be used. Neither of us like it, but I don't think we should not document public stable things just because we wish they were not in use. We can add some sort of deprecation separately.